cmd/cmd.go

```diff
@@ -36,7 +36,7 @@ var (
 // 2. Create database (if schemaOnly is set to false)
 // 3. Run data conversion (if schemaOnly is set to false)
 // 4. Generate report
-func CommandLine(driver, projectID, instanceID, dbName string, dataOnly, schemaOnly, skipForeignKeys bool, schemaSampleSize int64, sessionJSON string, ioHelper *conversion.IOStreams, outputFilePrefix string, now time.Time) error {
+func CommandLine(driver, projectID, instanceID, dbName string, dataOnly, schemaOnly, skipForeignKeys bool, schemaSampleSize int64, sessionJSON string, ioHelper *conversion.IOStreams, outputFilePrefix string, now time.Time, targetDb string) error {
```

Comment:
I'm wondering if the new targetDb parameter should be earlier in the list, rather than last parameter.
We could put it next to driver (since its the target counterpart), or maybe just after dbName.

I think go style is to use targetDB rather than targetDb e.g. like projectID.
We're probably not super-consistent on this.

Reply:
Agreed. I was also thinking it makes more sense being earlier. Will make the change